tests/fixtures/Search.2.0.ethiopic.json

```diff
@@ -99,6 +99,7 @@
             "match": {
                 "name": "\u1275\u130d\u122b\u12ed\u1275",
                 "type": "language",
+                "tag": "tig",
```

Comment:
Notice that the tag matches the definition from the package, rather than the base tag. This is intentional as we need to be providing the tag that the package knows about to the apps.